pymongo/mongo_client.py

```diff
@@ -503,6 +507,10 @@ def __init__(
            arguments.
            The default for `uuidRepresentation` was changed from
            ``pythonLegacy`` to ``unspecified``.
+           There is a new parameter for SRV URIs ``srvServiceName`` which
+           can be used like so::
+
+                MongoClient("mongodb+srv://test22.test.build.10gen.cc/?srvServiceName=customname")
```

Comment:
We should not repeat the docs here. We should just say:
```
Added the ``srvServiceName`` URI and keyword argument.
```

Reply:
Done.